src/libAtomVM/defaultatoms.c

```diff
@@ -184,6 +187,9 @@ void defaultatoms_init(GlobalContext *glb)
     ok &= globalcontext_insert_atom(glb, infinity_atom) == INFINITY_ATOM_INDEX;
     ok &= globalcontext_insert_atom(glb, timeout_value_atom) == TIMEOUT_VALUE_ATOM_INDEX;
 
+    ok &= globalcontext_insert_atom(glb, machine_atom) == MACHINE_ATOM_INDEX;
+    ok &= globalcontext_insert_atom(glb, avm_floatsize_atom) == AVM_FLOATSIZE_ATOM_INDEX;
+
```

Comment:
This is fine for now, but in the future we want to remove these constants and atom registrations, since they fill up the atoms table unnecessarily and consume memory.